src/classes/sendEmail.ts

```diff
@@ -8,7 +8,37 @@ export {
   SendMailOptions as SendEmailOptions,
   SentMessageInfo as SendEmailResult,
```

Comment:
I think we also want to move these 2 types to the `types.ts` file and export from there. They will need to be imported for use in this file as well.